src/modules/colorPicker/ColorPickerUI/Mouse/MouseInfoProvider.cs

```diff
@@ -111,6 +112,23 @@ private static System.Windows.Point GetCursorPosition()
             return (System.Windows.Point)lpPoint;
         }
 
+        private static double GetMainDisplayRefreshRate()
+        {
+            double refreshRate = 60.0;
+
+            foreach (var monitor in MonitorResolutionHelper.AllMonitors)
+            {
+                if (monitor.IsPrimary)
+                {
+                    EnumDisplaySettingsW(monitor.Name, ENUM_CURRENT_SETTINGS, out DEVMODEW lpDevMode);
```

Comment:
this returns bool, make sure it returns true before assigning `refreshRate` variable below